data/__generated__/schema.graphql

```diff
@@ -1,35 +1,35 @@
 scalar ColorHex
 
 input CreateIogLabelInput {
+  centerPoint: [Float!]!
```

Comment:
Has this file changed because you formatted it ? If so, won't it change every time the schema is generated ?

Reply:
Woah very nice catch 👁️🦅 !!!
However, after some investigation, I've figured out that the changes came from the upgrade of graphql-codegen which now sorts properties alphabetically by default. So it should not pollute the diff next time the schema will be generated.
However, once this PR gets merged, we'll have to call `codegen` again on the other opened PR when they have changed the schema.